File: Level1/문자열내맘대로정렬하기/solution.cpp
#include <string>
#include <vector>
#include <algorithm>


using namespace std;

vector<string> solution(vector<string> strings, int n) {
    vector<string> answer;
    vector<string> new_strings;

    for(int i = 0; i < strings.size(); i++){
        new_strings.push_back(strings[i][n] + strings[i]);
    }

    sort(new_strings.begin(), new_strings.end());

    for(int i = 0; i < new_strings.size(); i++){
        answer.push_back(new_strings[i].substr(1));
    }


    return answer;
}

File: Level1/문자열내맘대로정렬하기/solution.py
def solution(strings, n):
    answer = []
    key = []

    for i in range(len(strings)):
        strings[i] = strings[i][n] + strings[i]

    strings = sorted(strings)

    for i in range(len(strings)):
        strings[i] = strings[i][1:]

    return strings